test/azul_test_case.py

```diff
@@ -316,9 +323,15 @@ def dummy_get_credentials(_self):
             # These must match what `moto` uses to mock the instance metadata
             # response (see InstanceMetadataResponse.metadata_response() in
             # moto.instance_metadata.responses).
-            return Credentials(access_key='test-key',
-                               secret_key='test-secret-key',
-                               token='test-session-token')
+            credentials = Credentials(access_key='test-key',
+                                      secret_key='test-secret-key',
+                                      token='test-session-token')
+            # Unlike DeferredRefreshableCredentials, these dummy credentials
+            # lack an expiration date, so we add one in here. This is a required
+            # since when we generate a presigned URL, we check to see if the
+            # current credentials will expire before the presigned URL will.
```

Comment:
Typo, and I think it can expressed more succinctly.
```suggestion
            # lack an expiration date, so we add one in here to facilitate our
            # verification of the expiration date when creating presigned URLs.
```